test/specs/test_wishlist_page.js

```diff
@@ -0,0 +1,123 @@
+                Wishlist.wishlist_notification_close_icon.click()
+                Wishlist.open_wishlist_page()
+                expect(Wishlist.number_of_item_in_wishlist).toHaveText('(' + (product_quantity[1] + product_quantity[0]) + ')')
+                expect(Wishlist.get_sku_value(1)).toHaveText(products.products[1].sku)
+                expect(Wishlist.get_product_name(1)).toHaveText(products.products[1].name)
+                expect(Wishlist.get_price(1)).toHaveText(money_formatter.usd_format.format(products.products[1].price))
+                expect(Wishlist.get_total_value(1)).toHaveText(money_formatter.usd_format.format(products.products[1].price*product_quantity[1]))
+                expect(Wishlist.get_quantity_of_item(1)).toHaveAttribute('value', product_quantity[1])
+            })
+        })
+    
+        describe('Add different products to wishlist', () => {
+            it('17. should new products are showed correctly in wishlist', () => {
+                Wishlist.add_item_to_wishlist(products.products[2].name)
+                product_quantity.push(1)
+                expect(Wishlist.wishlist_notification_message).toHaveText("The product has been added to your wishlist")
+                Wishlist.wishlist_notification_close_icon.click()
+                Wishlist.open_wishlist_page()
+                let total_of_product = 0
+                for (var i = 0; i < product_quantity.length; i++) {
+                    total_of_product += product_quantity[i]
+                }
+                expect(Wishlist.number_of_item_in_wishlist).toHaveText('(' + total_of_product + ')')
+                assert(Wishlist.wishlist_records.length, product_quantity.length)
+                for (var i = 0; i < product_quantity.length; i++) {
+                    expect(Wishlist.get_sku_value(i)).toHaveText(products.products[i].sku)
+                    expect(Wishlist.get_product_name(i)).toHaveText(products.products[i].name)
+                    expect(Wishlist.get_price(i)).toHaveText(money_formatter.usd_format.format(products.products[i].price))
+                    expect(Wishlist.get_total_value(i)).toHaveText(money_formatter.usd_format.format(products.products[i].price*product_quantity[i]))
+                    expect(Wishlist.get_quantity_of_item(i)).toHaveAttribute('value', product_quantity[i])
+                }
+            })
+        })
+    
+        describe('Update quantity of product in wishlist page', () => {
+            it('18. should change quantity of product when clicking on update wishlist button', () => {
+                Wishlist.update_item_to_wishlist()
+                expect(Wishlist.get_quantity_of_item(1)).toHaveAttribute('value', 10)
+            })
+        })
+    
+        describe('Add to cart', () => {
+            it('19. should not add any products into shopping cart when unclick on the checkbox', () => {
```

Comment:
reword description to "should not add any products into shopping cart when no product have been checked"